client/ayon_blender/api/lib.py

```diff
@@ -621,6 +621,8 @@ def strip_namespace(containers):
             children = node.children_recursive
         elif isinstance(node, bpy.types.Object):
             children = node.children
+        elif isinstance(node, bpy.types.ShaderNodeTexImage):
```

Comment:
Is there's a reason to not just support the more generic base class `ShaderNode` or even `Node`?